spec/variables.md

```diff
@@ -9,19 +9,20 @@
 * [Semantics](#semantics)
   * [Executing a Variable Declaration](#executing-a-variable-declaration)
   * [Evaluating a Variable](#evaluating-a-variable)
-  
+
 ## Syntax
 
 <x><pre>
-**Variable**            ::= '$' Identifier | NamespacedVariable
+**Variable**            ::= VariableName | NamespacedVariable
+**VariableName**        ::= '$' Identifier
```

Comment:
To make the difference clearer, I think this should probably be something like `NamespacelessVariable` or `LocalVariable`.

Reply:
After writing this, I saw that https://github.com/sass/sass/pull/2850#discussion_r432189807 suggested `PlainVariable`.

`LocalVariable` seems wrong to me, as it could be the name of a global variable.

Reply:
I will let you decide which name I should use before updating my PR.